apps/aeoracle/src/aeo_utils.erl

```diff
@@ -8,50 +8,28 @@
 -module(aeo_utils).
 
 -export([check_format/3,
-         check_ttl_fee/3,
          check_vm_version/1,
          ttl_delta/2,
-         ttl_expiry/2,
-         ttl_fee/2]).
+         ttl_expiry/2
+        ]).
 
 -include_lib("apps/aecontract/src/aecontract.hrl").
 
--define(ORACLE_TTL_FEE, {1, 1000}). %% One part over a thousand.
+-spec ttl_delta(aec_blocks:height(), aeo_oracles:ttl()) ->
+    non_neg_integer() | {error, too_low_height}.
+ttl_delta(_CurrHeight, {delta, D}) -> D;
+ttl_delta(CurrHeight, {block, H}) when H > CurrHeight -> H - CurrHeight;
+ttl_delta(_CurrHeight, {block, _H}) -> {error, too_low_height}.
```

Comment:
I see semantics changed from exception `error({too_low_height, H, CurrHeight})` to value `{error, too_low_height}` - I need to check code around this.

Reply:
I see caller `aec_query_tx:check_oracle_ttl` calls this then performs sum:
```
        Delta  = aeo_utils:ttl_delta(Height, QTTL),
        MaxTTL = aeo_utils:ttl_expiry(Height + Delta, RTTL),
```
that means that this would raise badarith though it is inside `try ... catch _:_ ` so this would be caught and return `{error, invalid_ttl}` that sounds reasonable. Ok.

I see test code in `aeo_txs_eqc` uses `+` on result that shall result in runtime error anyway, and I see no logic on type of runtime error (error exception too_low_height vs. badarith exception) so look ok cc @hanssv .